app/public/profile.html

```diff
@@ -46,65 +49,83 @@ <h2>Edit</h2>
   if (token) {
     localStorage.setItem("token", token);
 
-    fetch('./api/v1/profile', {
-      credentials: 'include',
+    fetch("./api/v1/profile", {
+      credentials: "include",
```

Comment:
curious, what does this do? `credentials: "include"`

Reply:
Cookie?

Reply:
By default, fetch won't send or receive any cookies from the server, resulting in unauthenticated requests if the site relies on maintaining a user session (to send cookies, the credentials init option must be set).

Reply:
It is just a "test" page, in general consider using 'same-origin' instead of 'include'.

Reply:
In effect it was needed by Shibboleth, now we can get rid of it.